README_MACOS.md

```diff
@@ -295,6 +295,58 @@ This should be fixed at some point (its a build tool configuration issue). Until
 
 They do however work on Linux and Windows.
 
+Building with native JACK backend
+------------------------------
+
+If you want to use `scsynth` or `supernova` on macOS with JACK, but _without_ the JackRouter driver, you can build them with the native JACK backend. **In that case you will not be able to boot `scsynth`/`supernova` without booting JACK first.** JackRouter allows any macOS application to stream audio to/from JACK, but at the time of writing this (early 2019) it is outdated and the development has stalled.
+
+First you need to install JACK, either through `homebrew`:
+
+```
+brew install jack qjackctl
+```
+
+or by installing the `JackOSX` package. Please note, JACK from `homebrew` is `jack1` and does _not_ include JackRouter. `JackOSX` is `jack2` and it does include JackRouter. `jack1` and `jack2` implement the same API, but you should have only one of them installed at a time.
+
+In order to build with JACK, you need to add the `-DAUDIOAPI=jack` flag to cmake. The full command to configure the build (with supernova) would be:
+
+```
+cmake -G Xcode -DCMAKE_PREFIX_PATH=`brew --prefix qt5` -DSUPERNOVA=ON -DAUDIOAPI=jack ..
+```
+
+Afterwards proceed with the build process as usual.
+
+##### Running SuperCollider with JACK
+
+When `jack` is installed via `homebrew`, you need to add jack's path to the `$PATH` environment variable in `sclang`:
```

Comment:
> for instance, could we tell the servers via an env var or command line arg where the JACK executable is?

I'm not aware how to do this differently...

> same with setting SC_JACK_DEFAULT_INPUTS below. all of this could be made simpler and wrapped inside a helper function on ServerOptions. my question there is, do people want to use SC_JACK_DEFAULT_INPUTS=system by default? what do they gain by doing so?

`SC_JACK_DEFAULTS_INPUTS` is _I think_ the way you would do that on linux anyway... AFAIR this connects SC inputs/outputs to the soundcard. Otherwise, SC starts unconnected, so no sound is coming to the speakers (only useful if you are manually routing it to another app).

Reply:
i am asking you to try to think of a different way of doing it so people aren't manually managing environment variables.

Reply:
Managing of JACK connections is explained this way in the [help](https://github.com/supercollider/supercollider/blob/3e44dfe65d39a3e180f0a4203fc7b2cd3c8bfc35/HelpSource/Reference/AudioDeviceSelection.schelp#L90). Do you think I should come up with a different way to do this? If I do, then this Readme is probably not the best place to do it. My point reiterating info from the help here is to get user up and running quicker, but I'd be also fine pointing them to the help.

For adding jack executable to the $PATH, now I need to go back and revisit why did I have to do it in the first place.